frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -193,6 +206,30 @@ export const PaymentInput = (): JSX.Element => {
 
         {watchedEnabled && (
           <>
+            <FormControl
+              isReadOnly={paymentsMutation.isLoading}
+              isInvalid={!!errors.description}
+              isRequired
+            >
+              <Flex>
+                <FormLabel>Description</FormLabel>
+                <Spacer />
+                <Tooltip
+                  label={DESCRIPTION_TOOLTIP}
+                  placement="top"
+                  textAlign="center"
+                >
+                  <Icon as={BxsHelpCircle} aria-hidden ml="0.5rem" />
+                </Tooltip>
```

Comment:
actually just out of curiosity, why are we using a tooltip? could we be using the description field for form labels instead a la this -> 

<img width="527" alt="image" src="https://user-images.githubusercontent.com/25571626/228121197-ec416f13-677f-427d-b5e3-ed9f2fb2b942.png">

Reply:
That does make more sense, @staceytan1998 what do you think?

Reply:
Ok, will change the design! Thanks for bringing it up @justynoh !

Reply:
Thanks for the suggestion @justynoh, [5ec9794](https://github.com/opengovsg/FormSG/pull/5998/commits/5ec9794af2dde44085a7d7444327e025c11cf38a)